app/views/returns/ImportedPlasticPackagingWeightView.scala.html

```diff
@@ -64,11 +63,11 @@
         @govukInput(
             InputViewModel(
                 field = form("value"),
-                label = LabelViewModel(messages("importedPlasticPackagingWeight.heading")).asPageHeading()
+                label = Label()
```

Comment:
yes this parameter can be omitted completely as inputViewModel should now set that as default